ci/helm/environments/preview.yaml

```diff
@@ -1,11 +1,16 @@
-# enabled conditions
+# these values are required by helmfile-base.yaml.gotmpl releases
 nuxeo:
   enabled: true
 mongodb:
   enabled: true
+postgresql:
+  enabled: false
 opensearch:
   enabled: true
+kafka:
+  enabled: false
```

Comment:
By the way, do we want to add Kafka for the preview, to be consistent about the usual production stack: MongoDB, OpenSearch, Kafka?